### PR TITLE
Fixes pipe painter/RPD ignoring can_be_coloured

### PR DESCRIPTION
**Pull request by jknpj**
The more you know...

:cl:
 * rscadd: Fixes pipe painter/RPD being able to paint atmos devices that aren't supposed to be painted

### CODE REVIEW

**Comment by brndd** [comment]
wtf i cant paint the TEG anymore????? you are RUINING ENGINEERING [feature loss] [controversial] [powercreep] [do not merge] [can not reproduce]

**Comment by Kurfursten** [comment]
Okay but this *will* lead to the sad loss of being able to paint cryo tubes which was great

**Comment by beesil** [comment]
BLACKED.COM-ing the cryo tube is a FEATURE.

**Comment by Exxion** [comment]
You can just change `can_be_coloured` to `TRUE` for them

File: code/ATMOSPHERICS/pipes.dm [hunk 1]
```diff
@@ -27,10 +27,9 @@
 		var/datum/pipeline/pipeline = parent
 		var/list/update_later = list()
 		for(var/obj/machinery/atmospherics/pipe in pipeline.members)
-			pipe.color = mass_colour
 			if(!pipe.can_be_coloured)
-				pipe.default_colour = mass_colour
-				update_later += pipe
+				continue
+			pipe.color = mass_colour
```
Comment (Exxion) [inline code/ATMOSPHERICS/pipes.dm]: Seems simpler to just invert the `if()` and move this in there

File: code/ATMOSPHERICS/pipes.dm [hunk 2]
```diff
@@ -27,10 +27,8 @@
 		var/datum/pipeline/pipeline = parent
 		var/list/update_later = list()
 		for(var/obj/machinery/atmospherics/pipe in pipeline.members)
-			pipe.color = mass_colour
-			if(!pipe.can_be_coloured)
-				pipe.default_colour = mass_colour
-				update_later += pipe
```
Comment (Exxion) [inline code/ATMOSPHERICS/pipes.dm:33]: By the way, what the fuck was whoever wrote this thinking?

Comment (jknpj) [reply code/ATMOSPHERICS/pipes.dm:33]: :woman_shrugging:

**Pull request merged**